dist/build.gradle

```diff
@@ -90,8 +90,8 @@ task copyLib(group: 'Build',
         FileCollection jgits = layout.files { libDir.listFiles() }.filter { File file ->
             file.name.contains("jgit")
         }
-        // org.eclipse.jgit-6.x and org.eclipse.jgit.ssh.jsch-6.x are required.
-        if (jgits.size() != 2 || !jgits.any { File file -> file.name.startsWith("org.eclipse.jgit-6") }) {
+        // org.eclipse.jgit-6.x is required.
+        if (jgits.size() != 1 || !jgits.first().name.startsWith("org.eclipse.jgit-6")) {
             throw new IllegalStateException("Unexpected JGit JARs: $jgits.files")
```

Comment:
By the way, could we move `startsWith("org.eclipse.jgit-6")` to `FileCollection`'s filter and check only if the size of `jgits` is one?

Reply:
Ah, it seems intentionally used for leaving an exception message for the jar files.
Please ignore the commend above.